geps/gep-3171/index.md

```diff
@@ -11,7 +11,11 @@ Enhance the existing [Request Mirroring](https://gateway-api.sigs.k8s.io/guides/
 
 ## Goals
 
-Successfully implement the feature.
+Enable percentage-based request mirroring with Gateway and Gateway for mesh APIs.
+
+## Scope
+
+The scope of this GEP is to add support for this feature in both HTTPRoute and GRPCRoute
```

Comment:
If GRPCRoute is in scope (seems like a nice goal), I think we'd also need a separate conformance test and supported feature to cover that.

Reply:
probably worth iterating on it. I think it will be smarter to add GRPC route conformance to all kinds of mirror tests

Reply:
added an issue as well